files/build_templates/sonic_debian_extension.j2

```diff
@@ -516,10 +516,10 @@ sudo LANG=C cp $SCRIPTS_DIR/sonic-netns-exec $FILESYSTEM_ROOT/usr/bin/sonic-netn
 # Copy systemd timer configuration
 # It implements delayed start of services
 sudo cp $BUILD_TEMPLATES/snmp.timer $FILESYSTEM_ROOT_USR_LIB_SYSTEMD_SYSTEM
-sudo LANG=C chroot $FILESYSTEM_ROOT systemctl enable snmp.timer
+echo "snmp.timer" | sudo tee -a $GENERATED_SERVICE_FILE
 {% if enable_system_telemetry == 'y' %}
 sudo cp $BUILD_TEMPLATES/telemetry.timer $FILESYSTEM_ROOT_USR_LIB_SYSTEMD_SYSTEM
-sudo LANG=C chroot $FILESYSTEM_ROOT systemctl enable telemetry.timer
+echo "telemetry.timer" | sudo tee -a $GENERATED_SERVICE_FILE
```

Comment:
Why no longer calling `systemctl enable` on the timers?

Reply:
systemctl enable will create a mapping of the service in /etc/systemd/system/*.wants directory. This can cause issue in multi-asic platform. The dependencies that are created in systemd generator are present in /var/run/systemd/generator/*.wants directory. 
To explain better, I have directory structure from VS platform, with systemctl enable and without it.
**Without** enable: single asic VS platform
```
admin@sonic:**/var/run/systemd/generator**$ ls timers.target.wants/
snmp.timer  telemetry.timer
admin@sonic:**/var/run/systemd/generator**$ ls swss.service.wants/
nat.service  snmp.timer
admin@sonic:/etc/systemd/system/timers.target.wants$ ls
apt-daily.timer          logrotate.timer
apt-daily-upgrade.timer  process-reboot-cause.timer
admin@sonic:/var/run/systemd/generator/swss.service.wants$ 
.timer
â— snmp.timer - Delays snmp container until SONiC has started
   Loaded: loaded (/lib/systemd/system/snmp.timer; **enabled-runtime;** vendor prese
   Active: inactive (dead) since Mon 2020-06-22 00:35:35 UTC; 4min 40s ago
multi-asic:

admin@sonic:**/var/run/systemd/generator$ ls swss@0.service.wants/**
nat.service  snmp.timer
admin@sonic:/var/run/systemd/generator$ ls timers.target.wants/
snmp.timer  telemetry.timer
```
Though timers.target.wants directory is present in /etc/systemd/system and /var/run/systemd/generator, systemctl -r shows all under timers.target:
```
timers.target             loaded active     active          Timers      
---------------------------------------------------------------------------------       
apt-daily-upgrade.timer   loaded active     waiting         Daily apt upgrade a
apt-daily.timer           loaded active     waiting         Daily apt download 
fstrim.timer              loaded active     waiting         Discard unused bloc
logrotate.timer           loaded active     waiting         Daily rotation of l
process-reboot-cause.timer loaded active     waiting         Delays process-rebo
snmp.timer                loaded active     waiting         Delays snmp contain
systemd-tmpfiles-clean.timer loaded active     waiting         Daily Cleanup of 
telemetry.timer           loaded active     waiting         Delays telemetry co
```
If we keep sysctl enable, then in mutli-asic platform, /etc/systemd/system/swss.service.wants directory is created with snmp.timer in it. /var/run/systemd/generator/swss@0.service.wants also has snmp.timer. So, this is not inline with the required dependency.
```
admin@sonic:/etc/systemd/system/timers.target.wants$ ls
apt-daily.timer          logrotate.timer             snmp.timer
apt-daily-upgrade.timer  process-reboot-cause.timer  telemetry.timer
admin@sonic:/etc/systemd/system/swss.service.wants$ ls
snmp.timer
```